### DISCUSSION TITLE
Is there up to date documentation of the NUnit XML result file?

### DISCUSSION THREAD

**Discussion by nohwnd** [nunit/nunit#3879] [Q&A] [answered]
https://docs.nunit.org/articles/nunit/technical-notes/usage/XML-Formats.html
https://docs.nunit.org/articles/nunit/technical-notes/usage/Test-Result-XML-Format.html

Is there any documentation of the XML that is more up to date? I am planning to update the output of Pester to newer NUnit format (uses 2.5 right now) and I am looking for the most up to date documentation. Thanks!

**Comment by mikkelbu** [comment, marked as answer]
Hi @nohwnd 
As far as I know https://docs.nunit.org/articles/nunit/technical-notes/usage/Test-Result-XML-Format.html is the most up-2-date. Also note that there are some descriptive XML schemas in https://github.com/nunit/nunit/tree/master/src/NUnitFramework/framework/Schemas

**Reply by fflaten** [reply]
Those links have conflicting information. Ex. the first link allows "project" test-suite type while the schema doesn't.

Also, are there any replacements for categories in 2.5? Label looks like a single-valued field.

**Reply by fflaten** [reply]
> Also, are there any replacements for categories in 2.5? Label looks like a single-valued field.

For future readers: nunit-console adds `[Category('Something')]` as a property, ex `<property name="Category" value="Something" />` 🙂

**Comment by rprouse** [comment]
The most up to date thing we have is the XSD files in the solution, but they are very hard to parse.
